features/support/object_model_pages/ivl/homepage/ivl_documents_page.rb

```diff
@@ -22,4 +22,8 @@ def self.local_mec_evidence_actions
   def self.view_history_option
     "//div[@class='selectric-scroll']/ul/li[contains(text(), 'View History')]"
   end
+
+  def self.verify_option
+    "//div[@class='selectric-scroll']/ul/li[contains(text(), 'Verify')]"
```

Comment:
please use a page object model or css selector here

do not assert on hard-coded english text

Reply:
removed xpath